charts/kubewarden-controller/values.yaml

```
@@ -300,7 +300,7 @@ policy-reporter:
     image:
       registry: ghcr.io
       repository: kyverno/policy-reporter-ui
-      tag: 2.4.3
+      tag: 2.4.4
```

```suggestion
      tag: 2.4.3
```

policy-reporter 3.5.0 ships with 2.4.3. Updatecli shouldn't bump this.